### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by x0wer**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,3 +1,73 @@
...
+    def __init__(self, km: float | int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Distance) -> Distance:
```
Comment (Dimosphen1) [inline app/main.py:14]: Indicate that `other` could be `int` and `float` in the type annotation here and in the other places as well

**Review by Dimosphen1 — CHANGES_REQUESTED** [review]
Several changes were requested

File: app/main.py (hunk 1 above)
Comment (x0wer) [reply app/main.py:14]: yeah i already did this. check this request https://github.com/mate-academy/py-distance-class/pull/1256/commits/2825128ee55f3ed378beb4bc4cb0a09a4000065f

Comment (Dimosphen1) [reply app/main.py:14]: You should do it in the `__add__`, `__iadd__` and other methods. Your changes should be represented in the current PR (pull request), not in a separate commit.

**Review by Dimosphen1 — CHANGES_REQUESTED** [review]
Several changes were requested

File: app/main.py (hunk 1 above)
Comment (x0wer) [reply app/main.py:14]: Understandable, will do it tomorrow

**Pull request closed without merging**